test/utils/init.ts

```diff
@@ -1,2 +1,3 @@
 import '@mui/monorepo/test/utils/init';
 import './licenseRelease';
+import './addChartAssertions';
```

Comment:
Chart? Or should it have been `Chai`? 🙈

Reply:
Nice catch, I'll fix that one :laughing: